openslides/utils/autoupdate.py

```diff
@@ -40,7 +40,7 @@ def ws_add_site(message):
             # Skip apps that do not implement get_startup_elements
             continue
         for collection in get_startup_elements():
-            output.extend(collection.as_autoupdate_for_user(message.user))
+            output.extend(collection.as_autoupdate_for_user(message.user.id))
```

Comment:
The docstrings are not wrong. They only don't tell about a feature, which they already have. But I have no problem, if you like to change the docstrings.

message.user is the django anonymous user. Of cause you can add some code to check for it and if anonymous user is enabled and user is the django anonymous user, change it to our anonymous user. But I like it, that the user id can be used. So in the future, it would be possible not to get the hole user object from the session, but only the user id. It should be possible to get read of one db query, when only the id is needed